starter-code/js/application.js

```diff
@@ -80,15 +84,21 @@ window.onload = function(){
       checkAnswer(document.getElementById("user-answer").value);
     }
   }
-  
+  document.getElementById("time-limit").value = timeBonus;
+
+  document.getElementById("time-limit").onkeyup =  function(e) {
+      timeBonus = parseInt(document.getElementById("time-limit").value);
+  }
 
   var slider = new Slider('#number-limit-input', {
     tooltip:"hide"
   });
 
-
-
   slider.on("slide", function(sliderValue) {
     document.getElementById("number-limit").textContent = sliderValue;
   });
+
+ audio = new Audio('audios/cash-register.mp3');
+
+
 }
```

Comment:
great job! If you want to keep working on it, you could try to check if the operation is correct when the user types something, without waiting for the enter key being pressed, but the overall is good!